docs/README.md

```diff
@@ -4,6 +4,8 @@ This page provides instructions on how to configure the LinkedIn
 authenticator and the WSO2 Identity Server using a sample app to 
 demonstrate authentication.
 
+> For WSO2 Identity Server 7.0.0 and later versions, please refer to the [updated documentation](is7/README.md) for the latest configuration process.
```

Comment:
Shall we keep the IS 7.0.0 onwards docs in the main readme and place the older version docs at an inner folder?

Reply:
fixed